test/parallel/xyk-pallet.API.pool.deactivatedPromotedPool.test.ts

```diff
@@ -35,7 +38,7 @@ let testUser1: User;
 let sudo: User;
 let token1: BN;
 let liquidityId: BN;
-const defaultCurrencyValue = new BN(250000);
+const defaultCurrencyValue = new BN(250000000);
```

Comment:
why this?

Reply:
OMG, that was one of the variants; I forgot to return back, fixed